source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -232,6 +232,7 @@ Start full backup. The response phase indicates that the backup is `"initializin
 You need to poll the backup until the phase is `"ready"`. 
 Once the backup is ready the response will include `<to_checkpoint_id>` which
 should be used as the `<from_checkpoint_id>` in the next incremental backup.
+When starting a full backup, the `backup_mode` for all the disks will be `full` (`backup_mode="full"`)
```

Comment:
We should explain first that we have a backup_mode attribute. Maybe:

   Once the backup is ready, the response include the following elements:
   - to_checkpoint_id - what it means...
   - disks - what is means...